charts/cosi/Chart.yaml

```diff
@@ -27,10 +27,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.0
+version: 0.3.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: 1.2.0-alpha
+appVersion: 0.3.0-alpha
```

Comment:
@AkshaySainiDell : Please re-verify this change

Reply:
This is correct. I have taken reference from pervious release : [cosi-0.1.2](https://github.com/dell/helm-charts/releases/tag/cosi-0.1.2) https://github.com/dell/helm-charts/blob/d6ed125b36e648e6b0a9a37f49c86cd759a9f914/charts/cosi/Chart.yaml#L36